packages/frontend-core/src/components/grid/cells/RelationshipCell.svelte

```diff
@@ -248,7 +277,13 @@
     >
       {#each fieldValue || [] as relationship}
         {#if relationship[primaryDisplay] || relationship.primaryDisplay}
-          <div class="badge">
+          <div
+            class="badge"
+            class:extra-info={!!relationFields[relationship._id]}
```

Comment:
NAB/performance optimsation: Since svelte has no nice way to conditionally attach handlers, we could disable pointer events using CSS when the cell is not focused, which would improve performance since these mouseover/leave handlers wouldn't fire unless the cell is focused.

We could achieve that with something like
`class:disabled={!focused}`
and CSS of
```css
.values.disabled {
  pointer-events: none;
}
```

I would attach this to the `.values` div rather than each badge, since that encompasses all badges.